.github/CODEOWNERS

```diff
@@ -245,10 +245,20 @@
 # Cinnamon
 /pkgs/desktops/cinnamon @mkg20001
 
-#nim
-/pkgs/development/compilers/nim  @ehmry
-/pkgs/development/nim-packages  @ehmry
+# nim
+/pkgs/development/compilers/nim   @ehmry
+/pkgs/development/nim-packages    @ehmry
 /pkgs/top-level/nim-packages.nix  @ehmry
 
 # terraform providers
 /pkgs/applications/networking/cluster/terraform-providers @zowoq
+
+# piegames' corner
```

Comment:
You are removing other people from codeowners for these files

Reply:
The "piegames' corner" was a joke anyways, it's actually "Matrix". But this is still a very good point.